src/pages/study-page/study-results-page/study-results-page.tsx

```diff
@@ -0,0 +1,129 @@
+      </div>
+      <hr />
+      {getLessonStats()}
+      <BubbleDivider
+        className="studied-cards-divider"
+        variantColor="dark"
+        variantType="divider"
+        label="studied cards"
+      />
+      {getLessonCards()}
+    </div>
+  );
+
+  function getNavButtons() {
+    return (
+      <div className="study-results-page-nav-buttons">
+        <Button size="medium" onClick={() => navigate(`${paths.study}/${deck.metaData.id}`)}>
+          study again
+        </Button>
+        <Button size="medium" onClick={() => navigate(`${paths.deck}/${deck.metaData.id}`)}>
+          view deck
+        </Button>
+      </div>
+    );
+  }
+
+  function getLessonStats() {
+    const numCardsStudied = lessonCards.length;
+    const totalDeckProgress = '57%'; // TODO: need to decide how we will calculate deck progress and where it will be calculated
+    const totalTimeSpent = getFormattedMilliseconds(lessonTime);
+
+    return (
+      <>
+        <div className="lesson-stat">
+          <div className="lesson-stat-label">number of cards studied</div>
+          <div className="lesson-stat-result">
+            <div className="lesson-stat-number">{numCardsStudied}</div>
+            <CardStackIcon className="lesson-stat-icon" />
+          </div>
+        </div>
+        <div className="lesson-stat">
+          <div className="lesson-stat-label">total deck progress</div>
+          <div className="lesson-stat-result">
+            <div className="lesson-stat-number">{totalDeckProgress}</div>
+            <StarIcon className="lesson-stat-icon" />
+          </div>
+        </div>
+        <div className="lesson-stat">
+          <div className="lesson-stat-label">total time spent</div>
+          <div className="lesson-stat-result">
+            <div className="lesson-stat-number">{totalTimeSpent}</div>
+            <ClockIcon className="lesson-stat-icon" />
+          </div>
+        </div>
+      </>
+    );
+  }
+
+  function getLessonCards() {
+    const icons: Record<LessonCardOutcome, React.ReactNode> = {
+      correct: <CorrectIcon />,
+      incorrect: <IncorrectIcon />,
+      unseen: <SkippedIcon />,
+    };
+
+    const variants: Record<LessonCardOutcome, ReadOnlyFlashcardVariant> = {
+      correct: 'green',
+      incorrect: 'red',
+      unseen: 'light-blue',
+    };
+
+    return lessonCards.map((card) => {
+      const icon = icons[card.outcome];
+      const variant = variants[card.outcome];
+      return (
+        <div key={card.key} className="study-results-flashcard">
+          <div className="study-results-flashcard-icon">{icon}</div>
+          <ReadOnlyFlashcard
+            variant={variant}
+            frontText={card.front.text}
+            backText={card.back.text}
+            onFrontSpeakerClick={() => playLazyAudio(card.front.audio)}
+            onBackSpeakerClick={() => playLazyAudio(card.back.audio)}
+          />
+          <div className="study-results-flashcard-icon"></div>
```

Comment:
extra div?

Reply:
I was using this for spacing. Somehow I spaced using a variable and using a right-margin:joy: